tailor-app/src/main/java/com/future/tailormade/tailor_app/core/model/ui/order/OrderDesignUiModel.kt

```diff
@@ -0,0 +1,18 @@
+package com.future.tailormade.tailor_app.core.model.ui.order
+
+data class OrderDesignUiModel(
+
+    var color: String,
+
+    var discount: String? = null,
+
+    var id: String,
+
+    var image: String,
+
+    var price: String,
```

Comment:
how about in double? do it can easily calculate when discount applied

Reply:
Has been resolved